src/twisted/web/iweb.py

```diff
@@ -53,18 +53,26 @@ def getHeader(key):
         """
         Get an HTTP request header.
 
-        @type key: C{str}
+        @type key: C{bytes} or C{str}
```

Comment:
The [Twisted coding standard page](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html#docstrings) says that even references to Python's builtin types should use `L{}` instead of `C{}`.

Reply:
:+1: Good point, I was just mindlessly copying the existing style.